ui/webui/resources/BUILD.gn

```diff
@@ -34,20 +34,30 @@ group("grdp") {
   }
 }
 
-if (!is_android && !is_ios) {
+include_polymer = !is_android && !is_ios
+
+if (include_polymer) {
   build_webui("build") {
     grd_prefix = "brave"
 
     # Ensure all the files are relative to this dir (with a brave/ prefix),
     # and not //ui/webui/resources which is the default for build_cr_component.
-    if (!is_android && !is_ios) {
+    if (include_polymer) {
```

Comment:
I'm confused, what's the point of having `include_polymer` here if this entire target is wrapped in `include_polymer`? I think maybe you're pulling forward some redundant checks here

Reply:
Fixed in https://github.com/brave/brave-core/pull/26154/commits/b8754b2a9bd54be5fe0c2216ea7684b5bd523bad